### ISSUE TITLE
[IntelliJ][ESP][Mac] Pops up plugin error after install the 2019.2.develop.1292.11-12-2019.zip

### ISSUE THREAD

**Issue by v-lanjli** [microsoft/azure-tools-for-java#3732]
**Build:** https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1292/Azure/processDownloadRequest/artifacts/azure-toolkit-for-intellij-2019.2.develop.1292.11-12-2019.zip
**IntelliJ Version:** 2019.2
**OS:** Mac

**Repro Steps:**

1. Open IntelliJ IDEA.
2. Click on Configure, and show the drop down menu.
3. Click on Plugins.
4. Install Plugin  from disk.
5. Install the build.
6. Restart IDE.
7. Plugin Error pops-up.

**Results:** Plugin Error pops-up. Open IntelliJ again to check the installed plugin, the status of the Azure toolkit is disabled.
![image](https://user-images.githubusercontent.com/40199316/68660934-0e73bd00-0575-11ea-9550-fd48dbf17c54.png)
![image](https://user-images.githubusercontent.com/40199316/68660968-1a5f7f00-0575-11ea-8ca5-36b0e4462719.png)

Error Message:
Plugin 'com.microsoft.tooling.msservices.intellij.azure' failed to initialize and will be disabled.  Please restart IntelliJ IDEA.
```
java.lang.UnsupportedOperationException: View 'dos' not available
    at java.base/sun.nio.fs.AbstractFileSystemProvider.setAttribute(AbstractFileSystemProvider.java:80)
    at java.base/java.nio.file.Files.setAttribute(Files.java:1831)
    at com.microsoft.azuretools.authmanage.CommonSettings.initBaseDir(CommonSettings.java:163)
    at com.microsoft.azuretools.authmanage.CommonSettings.setUpEnvironment(CommonSettings.java:60)
    at com.microsoft.intellij.AzureActionsComponent.initAuthManage(AzureActionsComponent.java:138)
    at com.microsoft.intellij.AzureActionsComponent.initComponent(AzureActionsComponent.java:111)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:503)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:126)
    at com.intellij.openapi.application.impl.ApplicationImpl.lambda$load$7(ApplicationImpl.java:414)
    at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:169)
    at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:591)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:537)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:156)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:414)
    at com.intellij.idea.IdeaApplication.lambda$executeInitAppInEdt$5(IdeaApplication.java:150)
    at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
    at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
    at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
    at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
    at java.base/java.security.AccessController.doPrivileged(Native Method)
    at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
    at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:438)
    at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
    at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
    at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
    at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
    at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by Flanker32** [comment]
@v-lanjli This has been fixed in #3755, could you please help us verify this issue? Here is the bits: https://dev.azure.com/mseng/VSJava/_build/results?buildId=10924904

**Comment by v-lanjli** [comment]
Verified and fixed this issue. Closed this issue.
